packages/ods/src/components/checkbox/src/components/ods-checkbox/ods-checkbox.tsx

```diff
@@ -34,6 +34,7 @@ export class OdsCheckbox {
     }
     this.odsClear.emit();
     hasChange && this.onInput();
+    this.inputEl?.focus();
```

Comment:
did you try the behaviour when using multiples checkboxes with the same name?

Reply:
Yes, the focus zis set on the one where you call the clear